### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by rahul-321**

### CODE REVIEW

File: SparkJobs/src/main/scala/CassandraQueryEngine.scala [hunk 1]
```diff
@@ -53,9 +55,9 @@ class CassandraQueryEngine() extends Serializable {
 
 		spark.udf.register("BigMapIncrement",(map1:Map[String,Map[String, Int]], map2:Map[String,Map[String, Int]])=>Utilities.BigMapIncrement(map1:Map[String,Map[String, Int]], map2:Map[String,Map[String, Int]]))
 
-		spark.udf.register("MapIncrement",(map1:Map[String,Int], map2:Map[String,Int])=>Utilities.MapIncrement(map1:Map[String,Int], map2:Map[String,Int]))
+		spark.udf.register("MapIncrementWithoutKey",(map1:Map[String,Int], map2:Map[String,Int])=>Utilities.MapIncrementWithoutKey(map1:Map[String,Int], map2:Map[String,Int]))
```
Comment (raviteja-ms) [inline SparkJobs/src/main/scala/CassandraQueryEngine.scala]: You don't need to change the Utilities function name. Only need to change registered `udf` name.
You can overload it there.

File: SparkJobs/src/main/scala/Utilities.scala [hunk 2]
```diff
@@ -340,7 +341,7 @@ object Utilities{
 		return LabelValues
 	}
 	
-	def MapIncrement(myMap:Map[String,Int], key:String): Map[String,Int] = {
+	def MapIncrementWithKey(myMap:Map[String,Int], key:String): Map[String,Int] = {
```
Comment (raviteja-ms) [inline SparkJobs/src/main/scala/Utilities.scala]: You don't need to change the Utilities function name. You can overload it there

File: SparkJobs/src/main/scala/Utilities.scala [hunk 3]
```diff
@@ -419,10 +420,16 @@ object Utilities{
 		if (key_match.isEmpty) default else key_match.get.group(1)
 	}
 
-	def GetValueFromMap(mymap:Map[String,Int],key:String,default:Int) : Int = {
+	def GetIntFromMap(mymap:Map[String,Int],key:String,default:Int) : Int = {
+		return mymap.getOrElse(key,default)
+	}
+
+
+	def GetStringFromMap(mymap:Map[String,String],key:String,default:Int) : Any = {
```
Comment (raviteja-ms) [inline SparkJobs/src/main/scala/Utilities.scala]: How can default value be `Int` here..?? Change it to `String` and return type should be `String`

**Comment by rahul-321** [comment]
Sir I'll do it first thing in the morning.

**Pull request merged**